docs/src/components/DocLayout/index.tsx

```diff
@@ -102,97 +104,95 @@ export default function DocLayout({
-                          display="flex"
-                          align="end"
-                          justify={tabs && tabs.length > 0 ? "between" : "end"}
-                          tablet={{ maxWidth: "80%" }}
-                        >
-                          {tabs && <Tabs activeTab={location.pathname} tabs={tabs} />}
-                          {headerLink && (
-                            // align with tabs
-                            <Box padding={{ bottom: "XXSmall" }}>
-                              <HeaderLink href={headerLink} />
-                            </Box>
-                          )}
-                        </Box>
-                      )}
-                      <Grid
-                        columns="1fr"
-                        tablet={{ columns: `${tocHasItems ? "80% 20%" : "100%"}` }}
-                      >
-                        {tocHasItems && (
-                          <StyledTocWrapper>
-                            <Hide on={["smallMobile", "mediumMobile", "largeMobile"]}>{Toc}</Hide>
-                          </StyledTocWrapper>
+                <Hide block on={["smallMobile", "mediumMobile", "largeMobile", "tablet"]}>
+                  <StyledDocNavigationWidth>
+                    <StyledDocNavigationWrapper>
+                      <DocNavigation currentUrl={path} />
+                    </StyledDocNavigationWrapper>
+                  </StyledDocNavigationWidth>
+                </Hide>
+                <StyledMain>
+                  {custom ? (
+                    <div>{children}</div>
+                  ) : (
+                    <>
+                      {trail && <Breadcrumbs trail={trail} />}
+                      <Box padding={{ bottom: "XLarge" }}>
+                        <Stack inline align="center" spaceAfter="small">
+                          <AddBookmark />
+                          <Heading as="h1" type="display">
+                            {title}
+                          </Heading>
+                        </Stack>
+                        {description && (
+                          <Box padding={{ left: "XXLarge" }}>
+                            <Text>{description}</Text>
+                          </Box>
                         )}
-                        <StyledProse
-                          padding={
-                            noElevation
-                              ? { top: "none", bottom: "XLarge", left: "XLarge", right: "XLarge" }
-                              : "XLarge"
-                          }
-                          elevation={noElevation ? undefined : "raised"}
+                      </Box>
+                      <StyledMobileOutdent>
+                        {(tabs || headerLink) && (
+                          <Box
+                            display="flex"
+                            align="end"
+                            justify={tabs && tabs.length > 0 ? "between" : "end"}
+                            tablet={{ maxWidth: "80%" }}
```

Comment:
this component causes bug, with github icon, when layout changes, it's remains on 80%, when it should be on 100%. (When you click on guidelines)
![Screenshot 2021-05-13 at 11 52 04](https://user-images.githubusercontent.com/14054752/118109563-a7d99480-b3e1-11eb-8056-98294bcb611d.png)